### PR TITLE
chore: Log wallet name when wallet connect

### PR DESCRIPTION
**Pull request by Chef-Yogi**
### Summary
📈🌟🎨

Added wallet name tracking to Google Tag Manager events. Refactored `WalletModal.tsx` to improve code style and pass the wallet name to the `logGTMWalletConnectEvent` function.

> _We connect to the wallets of doom_
> _We track their names with `walletTitle`_
> _We send the data to the layer of fire_
> _We log the events with `logGTMWalletConnectEvent`_

### Walkthrough
*  Add `walletTitle` parameter to `logGTMWalletConnectEvent` function and `onWalletConnectCallBack` property to track wallet name in Google Tag Manager events ([link](https://github.com/pancakeswap/pancake-frontend/pull/8453/files?diff=unified&w=0#diff-07559817ac9fe511266cbdb35b165e001015b552d3ca401a73d598efaeeb5a3eL78-R78), [link](https://github.com/pancakeswap/pancake-frontend/pull/8453/files?diff=unified&w=0#diff-07559817ac9fe511266cbdb35b165e001015b552d3ca401a73d598efaeeb5a3eR84), [link](https://github.com/pancakeswap/pancake-frontend/pull/8453/files?diff=unified&w=0#diff-d4a85d21c19e0a0a9892411a3e5cb5c53a4a3fdb72b5afd8bd11cd0c00536903L62-R62), [link](https://github.com/pancakeswap/pancake-frontend/pull/8453/files?diff=unified&w=0#diff-d4a85d21c19e0a0a9892411a3e5cb5c53a4a3fdb72b5afd8bd11cd0c00536903L419-R424))
*  Move `onWalletConnectCallBack` function inside success block to avoid unnecessary calls and errors ([link](https://github.com/pancakeswap/pancake-frontend/pull/8453/files?diff=unified&w=0#diff-d4a85d21c19e0a0a9892411a3e5cb5c53a4a3fdb72b5afd8bd11cd0c00536903L419-R424))
*  Reorder imports in `WalletModal.tsx` file alphabetically for code style consistency ([link](https://github.com/pancakeswap/pancake-frontend/pull/8453/files?diff=unified&w=0#diff-d4a85d21c19e0a0a9892411a3e5cb5c53a4a3fdb72b5afd8bd11cd0c00536903L20-R26))

### CODE REVIEW

File: packages/ui-wallets/src/WalletModal.tsx [hunk 1]
```diff
@@ -416,11 +416,11 @@ export function WalletModalV2<T = unknown>(props: WalletModalV2Props<T>) {
...
             localStorage.setItem(walletLocalStorageKey, wallet.title)
-          }
-          try {
-            onWalletConnectCallBack?.()
-          } catch (e) {
-            console.error(e)
+            try {
+              onWalletConnectCallBack?.(wallet.title)
+            } catch (e) {
+              console.error(e)
```
Comment (ChefMomota) [inline packages/ui-wallets/src/WalletModal.tsx:422]: Can u add the `wallet.title` here easy for us to track the log.

**Review by chefjackson — DISMISSED** [review]

File: packages/ui-wallets/src/WalletModal.tsx (hunk 1 above)
Comment (ChefMomota) [reply packages/ui-wallets/src/WalletModal.tsx:422]: Yup. Wdty?
console.error(`${wallet.title},  ${e}`)

Comment (Chef-Yogi) [reply packages/ui-wallets/src/WalletModal.tsx:422]: sure

Comment (Chef-Yogi) [reply packages/ui-wallets/src/WalletModal.tsx]: Done

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**